app/controllers/v0/notice_of_disagreements_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module V0
+  class NoticeOfDisagreementsController < AppealsBaseController
+    private
+
+    def error_class(method:, exception_class:)
+      "#{self.class.name}##{method} exception #{exception_class} (NOD)"
```

Comment:
Just want to verify that the double # is intended.

Reply:
yep! it is. good 👁️